index.html

```diff
@@ -1 +1,980 @@
+                    </div>
+                    <span>Nov. 2</span>
+                </div>
+            </div>
+
+        </section>
+        <div class="article-top">
+            <p>Editors’ Picks</p>
+        </div>
+        <section class="footer-containers">
+            <div class="articleImage">
+                <div>
+                    <img src="assets/add1." alt="add image">
+                    <div>
+                        <p>How the Space Station Became a Base to Launch Humanity’s Future</p>
+                    </div>
+                    <span>Nov. 2</span>
+                </div>
+            </div>
+            <div class="articleImage">
+                <div>
+                    <img src="assets/add2.jpg" alt="add2 image">
+                    <div>
+                        <p>The International Space Station: 20 Years of Promise, Peril and Fun</p>
+                    </div>
+                    <span>10h ago</span>
+                </div>
+            </div>
+            <div class="articleImage">
+                <div>
+                    <img src="assets/add3.jpg" alt="add3 image">
+                    <div>
+                        <p>NASA’s Asteroid Mission Packs Away Its Cargo. Next Stop: Earth.</p>
+                    </div>
+                    <span>Nov. 2</span>
+                </div>
+            </div>
+        </section>
+    </article>
+    <div class="bottom-side-bar">
+        <h4>Most Popular</h4>
+        <div class="side-bar-margin">
+            <a href='#'>
+                <p> What’s Left in the Seven States That Will Decide the Race</p>
+            </a>
+            <a href='#'>
+                <p>Can Biden Still Win? Can Trump Still Win? Yes. Here Are the Remaining Paths.</p>
+            </a>
+            <a href='#'>
+                <p>The Remaining Vote in Pennsylvania Appears to Be Overwhelmingly for Biden</p>
+            </a>
+            <a href='#'>
+                <p>Lauren Boebert, Hard-Right Gun Activist, Wins in Colorado House District</p>
+            </a>
+            <a href='#'>
+                <p>Fox News Made a Big Call in Arizona, Buoying Biden and Angering Trump</p>
+            </a>
+            <a href='#'>
+                <p>$1.25 Million Homes in Maine, Texas and New York</p>
+            </a>
+            <a href='#'>
+                <p>In Voting Cases, Chief Justice Roberts Is Alone but in Control</p>
+            </a>
+            <a href='#'>
+                <p>Dementia ‘Took Its Toll’ on Sean Connery, Wife Says</p>
+            </a>
+            <a href='#'>
+                <p>What We Know About the Vote in Wisconsin, Michigan and Pennsylvania</p>
+            </a>
+            <a href='#'>
+                <p>When He Bought the Tiny House, He Was Single. That Didn’t Last.</p>
+            </a>
+        </div>
+    </div>
+    <br>
+    <br>
+    <br>
+    <div class="footer-border"></div>
```

Comment:
- [ ]  This kind of `<br>` patterns are not needed if you use margins and padding to create the surrounding white space. [OPTIONAL]

Reply:
Thank you for the above suggestion, we will try to follow from the next project